README.md

```diff
@@ -1,3 +1,37 @@
 # Grunt incuna plugins
 
 A set of useful build and testing plugins easily reusable across projects.
+
+* [Adding new plugins](#adding-new-plugins)
+* [Plugins](#plugins)
+ * [update-po-files](#update-po-files)
+
+## Plugins
+
+### `update-po-files`
+
+A plugin for automatically updating `.po` files in a given directory from a `template.pot` file in that same directory.
+Usage:
+```js
+'update-po-files': {
+    all: {
+        src: 'my-i18n-directory'
```

Comment:
So `update-po-files` should be a multi task, and you use [`this.options(someDefaults)`](http://gruntjs.com/api/inside-tasks#this.options) to get options for all targets combined with target-specific options and default options, and then [`this.files`](http://gruntjs.com/api/inside-tasks#this.files) to access all the defined files you want to work with.

Something like this would work well:
```js
'update-po-files': {
    options: {
        template: 'i18n/template.pot'
    },
    some: {
        files: ['i18/some/*.po']
    },
    other: {
        files: ['i18/other/*.po']
    }
}
```